static/app/views/replays/detail/layout/asideTabs_v2.tsx

```diff
@@ -0,0 +1,71 @@
+import React, {useState} from 'react';
+import styled from '@emotion/styled';
+
+import NavTabs from 'sentry/components/navTabs';
+import Placeholder from 'sentry/components/placeholder';
+import {useReplayContext} from 'sentry/components/replays/replayContext';
+import {t} from 'sentry/locale';
+import ReplayReader from 'sentry/utils/replays/replayReader';
+
+import TagPanel from '../tagPanel';
+
+import ResizePanel from './resizePanel';
+import {BreadCrumbsContainer, VideoContainer} from '.';
+
+type Props = {
+  showCrumbs?: boolean;
+  showVideo?: boolean;
+};
+
+const TABS = {
+  video: t('Replay'),
+  tags: t('Tags'),
+};
+
+function AsideTabsV2({showCrumbs = true, showVideo = true}: Props) {
+  const {replay} = useReplayContext();
+  const [active, setActive] = useState<string>('video');
```

Comment:
```suggestion
  const [active, setActive] = useState('video');
```

also we could replace `useState` with `useUrlParams()` so that the url reflects which tab the user has open.